resources/properties.json

```diff
@@ -17,5 +17,7 @@
   "serviceAdvisoriesWmataUrl": "/Incidents.svc/json/Incidents",
   "noServiceAdvisoriesText": "There are no service alerts at this time.",
   "noServiceAdvisoriesTitle": "No Service Alerts",
-  "noStationRequestedSpeechOutput": "I need to know where you're departing from. Try asking \"When does the next train arrive at Metro Center?\""
+  "noStationRequestedSpeechOutput": "I need to know where you're departing from. Try asking \"When does the next train arrive at Metro Center?\"",
+  "setHomeStationSpeechOutput": "Welcome to the Capital Transit Skill! Which station would you like to set as your home station? Say 'Set my home station to...'",
+  "setHomeStationRepromptSpeechOutput": "Which station would you like to set as your home station? Say 'Set home station to...'"
```

Comment:
Does the `...` at the end affect how the Echo pronounces it? I would think it would at least add a delay in the speech.